sha512/src/main/kotlin/org/komputing/khash/sha512/extensions/PublicExtensions.kt

```diff
@@ -0,0 +1,24 @@
+package org.komputing.khash.sha512.extensions
+
+import org.komputing.khash.sha512.Sha512
+
+/**
+ * Returns the SHA512 digest of this byte array.
+ */
+fun ByteArray.sha512() = Sha512.digest(this)
+
+/**
+ * Returns the SHA512 digest of this string.
+ */
+fun String.sha512() = this.toByteArray(Charsets.UTF_8).sha512()
+
+/**
+*
+*/
+fun LongArray.toHexString(): String {
```

Comment:
I think this can be removed - does not really belong here and seems to be unused. If needed it should live in KHash

Reply:
Indeed, old code. Is removed now